datagen/options.go

```diff
@@ -11,6 +11,7 @@ type General struct {
 
 // Connection struct that stores info on connection from command line args
 type Connection struct {
+	Uri            string `long:"uri" value-name:"<uri>" description:"mongodb uri to connect to e.g. mongodb://<user>:<pass>@<host>:<port>"`
```

Comment:
can you add the link to MongoDB String URI format ( https://docs.mongodb.com/manual/reference/connection-string/ ), and mention the fact that URI option will take precedence over the other options if both are specified ?